rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskController.java

```diff
@@ -158,6 +159,12 @@ public ResponseEntity<TaskSummaryPagedRepresentationModel> getTasks(
         TaskQueryGroupByParameter.class,
         QuerySortParameter.class,
         QueryPagingParameter.class);
+
+    if (QueryParamsValidator.hasQueryParameterValuesOrIsNotTrue(request, "owner-is-null")) {
+      throw new InvalidArgumentException(
+          "It is prohibited to use the param owner-is-null with values.");
```

Comment:
Can you state here that "owner-is-null=true" should be allowed

Reply:
I don't think we should do that, because "owner-is-null=true" is not officially supported, right @holgerhagen @gitgoodjhe ?

Reply:
That's correct, we only tolerate it ;)